ISISComputingGroup/EPICS-ioc
# Ticket2026: Parallel DAE begin/end

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/2026

### To test

* Create and start a simple IOC with BEGINRUN and ENDRUN PVs (to model the DAE3)
* Start the TEKAFG3XXX_01 in recsim (will have to do a caput to the SIM PV)
* Confirm that the macros in the DAE st.cmd are uncommented and pointing at the test IOC and the TEKAFG3XXX_01
* Start the DAE
* Begin and end runs, confirming the behaviour outlined in the ticket is achieved (note the veto requires two channels)

* Confirm that commenting out the macros in the st.cmd produces no change from normal behaviour on the DAE

---

#### Code Review


### Functional Tests


### Final steps
